From 64ea0093ff1dda477fb178ceab518ef5c1879118 Mon Sep 17 00:00:00 2001
From: Masahiro Yamada <masahiroy@kernel.org>
Date: Wed, 8 Mar 2023 20:52:37 +0900
Subject: [PATCH 3/9] scripts/mksysmap: remove comments described in nm(1)

I do not think we need to repeat what is written in 'man nm'.

Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
---
 scripts/mksysmap | 20 +-------------------
 1 file changed, 1 insertion(+), 19 deletions(-)

diff --git a/scripts/mksysmap b/scripts/mksysmap
index 16a08b8ef..fea65fc3b 100755
--- a/scripts/mksysmap
+++ b/scripts/mksysmap
@@ -9,25 +9,7 @@
 
 #####
 # Generate System.map (actual filename passed as second argument)
-
-# $NM produces the following output:
-# f0081e80 T alloc_vfsmnt
-
-#   The second row specify the type of the symbol:
-#   A = Absolute
-#   B = Uninitialised data (.bss)
-#   C = Common symbol
-#   D = Initialised data
-#   G = Initialised data for small objects
-#   I = Indirect reference to another symbol
-#   N = Debugging symbol
-#   R = Read only
-#   S = Uninitialised data for small objects
-#   T = Text code symbol
-#   U = Undefined symbol
-#   V = Weak symbol
-#   W = Weak symbol
-#   Corresponding small letters are local symbols
+# The following refers to the symbol type as per nm(1).
 
 # For System.map filter away:
 #   a - local absolute symbols
-- 
2.40.0.71.g950264636c

